qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -470,6 +470,49 @@ public void doesNotRejectUrlsInLongLines() throws Exception {
     }
 
     /**
+     * CheckstyleValidator can ensure that spaces can exist outside of methods
```

Comment:
@JimDeanSpivey *CheckstyleValidator can allow spaces between methods of anonymous classes.*
(http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names)